Mathlib/RingTheory/HahnSeries/Addition.lean

```diff
@@ -549,6 +549,20 @@ def embDomainLinearMap (f : Γ ↪o Γ') : HahnSeries Γ R →ₗ[R] HahnSeries
 
 end Domain
 
+variable (R) in
+/-- `HahnSeries.truncLT` as a linear map. -/
+def truncLTLinearMap [DecidableLT Γ] (c : Γ) : HahnSeries Γ V →ₗ[R] HahnSeries Γ V where
+  toFun := truncLT c
+  map_add' x y := by
```

Comment:
Can you prove these two results as separated lemmas (with minimal assumptions)? Thanks!